src/components/FormAlertWithSubmitButton.js

```diff
@@ -132,15 +69,13 @@ const FormAlertWithSubmitButton = (props) => {
                 isDisabled={props.isDisabled}
                 isLoading={props.isLoading}
             />
-        </View>
-    );
-};
+        )
+        )}
```

Comment:
This indenting looks strange. I think the `)` should be indented by 4 more spaces. As long as the linter doesn't complain